cmd/minikube/cmd/logs.go

```diff
@@ -52,6 +53,11 @@ var logsCmd = &cobra.Command{
 	Short: "Returns logs to debug a local Kubernetes cluster",
 	Long:  `Gets the logs of the running instance, used for debugging minikube, not user code.`,
 	Run: func(cmd *cobra.Command, args []string) {
+		if !isClusterRunning() {
```

Comment:
why bother checking if it is NOT running at all ?
how about we just output the audit and last start if it exists at all the time ? (without checking if there is a minikube config or not)

Reply:
We could do this, the only thing is the profile not found message will still be displayed, so it will look like the following:
```
==> Audit <==
...

==> Last Start <==
...
🤷  Profile "minikube" not found. Run "minikube profile list" to view all profiles.
👉  To start a cluster, run: "minikube start"
```
If that's something we're ok with then we could definitely output the logs without the check, the check only exists to make sure that message is not output.

Reply:
yeah I think that is acceptable

how about we add a Block header for it ?
==> Audit <==
...

==> Last Start <==
...
==> Internal Logs <==

Reply:
That makes sense if the cluster if not running, the only thing is if the cluster is running, there will be a double block header for the first internal log.

Example:
```
==> Audit <==
...

==> Last Start <==
...
==> Internal Logs <==
==> Docker <==
```

What's your thoughts on that?

Reply:
good question ! I think u are right, lets not have the "internal logs" header at all.